entities/card.py

```diff
@@ -1,32 +1,44 @@
 from dataclasses import dataclass
-from entities.card_type import CardType
+
+from flask import Request
+
 from entities.card_design_properties import CardDesignProperties
-from datetime import datetime
-from uuid import UUID
-from typing import Dict
-from card_type import CardType
+from typing import Dict, List
+from entities.card_proprties import CardProperties
+from consts import *
 
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
 
 
 @dataclass
 class Card:
-    content: str
-    type: CardType
+    card_properties: CardProperties
     design_properties: CardDesignProperties
-    date: datetime
-    version: str
-    id: UUID
 
-    def serialize_card(self) -> Dict[str, str]:
+    def card_to_dict(self) -> Dict[str, str]:
         serialized_card = {
-            "content": self.content,
-            "card_type": self.card_type.value,
-            "date": str(self.date),
-            "version": self.version,
-            "id": self.id
+            CONTENT: self.card_properties.content,
+            CARD_TYPE: self.card_properties.type.value,
+            DATE: str(self.card_properties.date),
+            VERSION: self.card_properties.version,
+            ID: self.card_properties.id
         }
         for key, value in serialized_card.items():
```

Comment:
Not sure this loop is nessecery it's ok to leave null on json. 
BTW after running `json.load` we get `Dict[str, Any]` so it might be relevant here.